protocol/0069-VCBS-validators_chosen_by_stake.md

```diff
@@ -383,6 +387,11 @@ Verify that 2 validators are replaced, one in each epoch
 
 Verify that one validator is replaced the following epoch, one in the epoch after
 
+13. Ersatzvalidator reward (<a name="0069-VCBS-051" href="#0069-VCBS-051">0069-VCBS-051</a>)
```

Comment:
in this case the antiwhaling has no meaning as if the demoted ersatz validator is the only ersatzvalidator, they take the whole lot allocated to ersatzvalidators. Depending on the value of network.validators.ersatz.rewardFactor this can definitely give them a higher reward than tendermint validators. It actually highlights another aspect of it, suppose you have 2 ersatzvalidators, one as in your example where it has 50% of the total delegation of tendermint validators - and another earsatz validator with half of the delegation of any other tendermint validator (so lets say all tendermint validators have 10k, and the downgraded ersatz validator has 40k, and the other ersatz validator has 5k) - assuming perfect performance - the split between the ersatzvalidators after antiwhaling will be 2:1 but the split of the reward pool will be 50:(45 * scale_down), so depending on how much we scale the ersatz delegation down, there's still a chance that the ersatz validators (even the small one with 5k delegation) will get paid more than the tendermint validators. 

Let's say scale_down=0.5 this gives the ersatz validators 22.5/72.5 to be split 2:1 between them, which gives the bigger validator (2/3)*22.5/72.5 = 0.2068965517 of the total reward which is still bigger share than any of the tendermint validators which would get 0.2 * 50/72.5 = 0.1379310345

Reply:
I fixed the AC; the other issue is something that'd require some redesign (or implementing parameter-dependencies), so I'd propose that's post-launch.

Reply:
(we already have implemented param dependency, but happy to wait till after launch)